tests/pds/backup_test.go

```diff
@@ -72,3 +142,13 @@ func deleteAllBkpTargets() {
 		log.FailOnError(err, "Failed while deleting the GCP backup target")
 	}
 }
+
+func randString(length int) string {
```

Comment:
This is a utility function so should go in utils

Reply:
Done